changelog/814.bugfix.rst

```diff
@@ -0,0 +1 @@
+Use cache for threads in interaction data instead of always creating a new instance.
```

Comment:
this has the potential to no longer exist if the thread was deleted since the interaction was created.

this also means that it won't necessarily reflect the data provided with the interaction

Reply:
misunderstood this, just means that it uses the cache on creation, not on access. This changelog entry may need to be updated.

Reply:
I've tried to clarify it in https://github.com/DisnakeDev/disnake/pull/814/commits/e9ee06649e7401d9883edbb6f8f4fb6c5231a72b, should be good now.